rpc/core/env.go

```diff
@@ -35,13 +36,23 @@ const (
 
 var (
 	// set by Node
-	env *Environment
+	mut       = &sync.Mutex{}
+	globalEnv *Environment
 )
 
-// SetEnvironment sets up the given Environment.
-// It will race if multiple Node call SetEnvironment.
+// SetEnvironment sets the global environment to e. The globalEnv var that this
+// function modifies is protected by a sync.Once so multiple calls within the
```

Comment:
[question] for @evan-forbes where does this sync.Once occur?

Reply:
I am not Evan 😄, but it seems like this comment is a relic that wasn't cleaned together with past sync.Once removal

Reply:
it was a comment from a previous iteration and should be removed